tests/test_logsource.py

```diff
@@ -53,7 +53,9 @@ def get_detection_field(self,detection: dict):
         def get_field_name(selection: dict):
             name = []
             for field in selection:
-                if "|" in field:
+                if field == '|all':
+                    continue
```

Comment:
Just to confirm. This is to handle the keyword rules right?

Reply:
Yes , otherwise get an error because of the empty name field.